web/src/LogActions.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from "react"
+import { useStorageState } from "react-storage-hooks"
+import styled from "styled-components"
+import { incr } from "./analytics"
+import ClearLogs from "./ClearLogs"
+import { Color, FontSize, mixinResetButtonStyle } from "./style-helpers"
+
+export const LogFontSizeScaleLocalStorageKey = "tilt.global.log-font-scale"
+export const LogFontSizeScaleCSSProperty = "--log-font-scale"
+export const LogFontSizeScaleMinimumPercentage = 10
+
+const LogActionsGroup = styled.div`
+  margin-left: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`
+
+const FontSizeControls = styled.div`
+  color: ${Color.gray6};
+  vertical-align: middle;
+  display: flex;
+  flex-wrap: nowrap;
+`
+
+const FontSizeControlsDivider = styled.div`
+  font-size: ${FontSize.default};
+  &:after {
+    content: "|";
+  }
+`
+
+const FontSizeButton = styled.button`
+  ${mixinResetButtonStyle}
+  color: ${Color.gray6};
+
+  padding: 0 4px;
+
+  &:after {
```

Comment:
ditto here - seems weird to put this in ':after' selector rather than plain html content